app/main/routes.py

```diff
@@ -636,6 +639,11 @@ def record(record_id: uuid.UUID):
 @access_token_sign_in_required
 def download_record(record_id: uuid.UUID):
     file = db.session.get(File, record_id)
+    ayr_user = AYRUser(session.get("user_groups"))
+    can_download_records = ayr_user.can_download_records
+
+    if can_download_records is False:
```

Comment:
I think `if not can_download_records` or even `if can_download_records is not True` might be safer, just in case it comes is null or otherwise for whatever reason